src/MSIExtract/Views/TableView.xaml

```diff
@@ -0,0 +1,44 @@
+﻿<UserControl x:Class="MSIExtract.Views.TableView"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" mc:Ignorable="d"
+             xmlns:Controls="clr-namespace:MSIExtract.Controls"
+             xmlns:Views="clr-namespace:MSIExtract.Views"
+             xmlns:Msi="clr-namespace:MSIExtract.Msi;assembly=MSIExtract.Core"
+             d:DesignHeight="450" d:DesignWidth="800" d:DataContext="{StaticResource DesignTimeAppModel}">
+    <Grid>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="2*" />
+            <ColumnDefinition Width="5" />
+            <ColumnDefinition Width="5*" />
+        </Grid.ColumnDefinitions>
+
+        <ListBox Grid.Column="0" x:Name="TableNamesListBox" ItemsSource="{Binding Path=Tables}" IsEnabled="{Binding IsMsiLoaded}" >
+            <ListBox.ItemTemplate>
+                <DataTemplate>
+                    <StackPanel Orientation="Horizontal">
+                        <TextBlock Text="{Binding Name}" >
+                            <TextBlock.Style>
+                                <Style TargetType="{x:Type TextBlock}">
+                                    <Style.Triggers>
+                                        <DataTrigger Binding="{Binding IsEmpty}" Value="True">
+                                            <Setter Property="Foreground" Value="{x:Static SystemColors.GrayTextBrush}"/>
+                                        </DataTrigger>
+                                    </Style.Triggers>
+                                </Style>
+                            </TextBlock.Style>
+                        </TextBlock>
+                    </StackPanel>
+                </DataTemplate>
+            </ListBox.ItemTemplate>
+        </ListBox>
+
+        <GridSplitter Grid.Column="1" Width="5" HorizontalAlignment="Stretch" />
+
+        <DataGrid Grid.Column="2" DataContext="{Binding ElementName=TableNamesListBox,Path=SelectedItem}" ItemsSource="{Binding Path=Rows}"
+                  AutoGenerateColumns="True" SelectionMode="Single" IsReadOnly="True" CanUserResizeRows="False" CanUserReorderColumns="False" HeadersVisibility="Column" SelectionUnit="Cell" CanUserAddRows="False" CanUserDeleteRows="False" />
```

Comment:
```suggestion
                  AutoGenerateColumns="True" SelectionMode="Single" IsReadOnly="True" CanUserResizeRows="False" CanUserReorderColumns="False" HeadersVisibility="Column" SelectionUnit="FullRow" CanUserAddRows="False" CanUserDeleteRows="False" />
```

Full-row selection will look much better.

Reply:
It might look better, but is less useful when copying data.
I could change selectionmode to allow multiple cells?

Reply:
You raise a good point re copying cell data. I did not think of that.